### PR TITLE
[DRAFT] chore: remove top-level deps as no longer used

### PR DESCRIPTION
**Pull request by mdjastrzebski**
### Summary

Root-level `package.json` contained `peerDeps` and `devDeps` which were no longer used, since we now have package deps.

### Test plan

All checks and test pass.

### CODE REVIEW

File: packages/reassure-measure/package.json [hunk 1]
```diff
@@ -38,8 +38,13 @@
   "dependencies": {
     "mathjs": "^10.1.1"
   },
+  "devDependencies": {
+    "@types/jest": "^28.0.0",
+    "typescript": "^4.5.2"
```
Comment (thymikee) [inline packages/reassure-measure/package.json:43]: I think we don't need to declare those per package, should be fine to only do it in root package.json?

File: packages/reassure-measure/package.json [hunk 2]
```diff
@@ -38,8 +38,13 @@
...
+  },
   "peerDependencies": {
-    "react": "*"
+    "react": "*",
+    "jest": ""
```
Comment (thymikee) [inline packages/reassure-measure/package.json:47]: it's not a peer dep, one can use vitest

**Comment by mdjastrzebski** [comment]
Closing as not really needed now.

File: packages/reassure-measure/package.json (hunk 2 above)
Comment (nikolasrieble) [reply packages/reassure-measure/package.json:48]: Can one use vitest? The README states that jest is required. 🤔

**Pull request closed without merging**